rawsnippets/pipelineResource.yaml

```diff
@@ -3,6 +3,6 @@ kind: PipelineResource
 metadata:
   name: ${1:git-source}
 spec:
-  type: ${2|git,image,pullRequest,cluster,storage|}
+  type: ${2|git,image,pullRequest,cluster,storage,gcs,build-gcs,cloudevent|}
```

Comment:
apiVersion should be `tekton.dev/v1beta1`

Reply:
No, according to this https://github.com/tektoncd/pipeline/blob/master/docs/migrating-v1alpha1-to-v1beta1.md#pipelineresources-and-catalog-tasks